### PR TITLE
✏️ update docs with bench critieria

### PR DESCRIPTION
**Pull request by migbro**
## Description

This adds our benchmarking story behind our filtering criteria for tumor-only calling

Fixes # https://d3b.atlassian.net/browse/BIXU-1928

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Below we attempt to concisely summarize the story and results.  
+
+### Test synthetic sample `syn_WGS_IL_N_1` HIVE filter components and final combination
+- RAW: the protected file as output by our pipeline
+- PASS: Stripped the public file soft filters from the RAW and grabbed only the PASS records
+- PASS+GNO 0.1%: Applied the KF gnomAD filter (exclude gnomad >= 0.001) directly to the PASS file. These results should be a pretty close approximation of what our public files look like
+- PASS+GNO 0.003%: Applied the HIVE gnomAD filter (exclude gnomad >= 0.00003) directly to the PASS file. I used this as a comparison to see just how much value we were getting from the other filtration steps
+- PASS+PON: Removed any records from the PASS file that were also found in the GATK PON
+- PASS+PON+dbSNP: Removed any records from PASS+PON that were labeled COMMON in dbSNP
+- PASS+PON+dbSNP+ClinVar: Removed any records from PASS+PON+dbSNP that had "benign" in their worst VEP assigned CLN_SIG
```
Comment (wongjessica93) [inline docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md:51]: This seems unclear to me what it means reading as someone who has not used this pipeline before: 
"had "benign" in their worst VEP assigned CLN_SIG"

**Review by wongjessica93 — APPROVED** [review]

File: docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+# Tumor-Only SNV Benchmarking Methods and Results
+When it comes to somatic variant calling, having a matched normal sample from the patient is always preferred.
+However, this is not always possible.
+Mutect2, in our opinion as of this writing, is the best caller to use when a matched normal is not available in tumor-only calling mode.
+Using the same gold standard dataset that we used to benchmark (see the first two sections of [this doc](https://github.com/kids-first/kf-somatic-workflow/blob/feature/mb-add-bench-doc/docs/SOMATIC_SNV_BENCHMARK.md)) our somatic with matched tumor-normal variant calling methods, we reviewed filtering strategies from other publications ([TOSCA](https://academic.oup.com/bioinformaticsadvances/article/2/1/vbac070/6717791?login=false) and [HIVE Dragen Pipeline](https://www.fda.gov/science-research/fda-science-forum/hive-dragen-pipeline-enables-somatic-tumor-only-filtration)), and chose the best combination that maximized F1 score while minimizing true positive (TP) loss as aggressively pursuing false positive (FP) reduction and results in a massive loss of TP calls.
```
Comment (dmiller15) [inline docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md:5]: Just making a note to update the link for `this doc` here. It's currently pointing to your branch which will be deleted. Now that you've merged we should be pointing to master.

File: docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md (hunk 1 above)
Comment (dmiller15) [reply docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md:51]: It's VEP jargon. `CLN_SIG` is the VEP key for "ClinVar clinical significance of the dbSNP variant". See [Default VEP output](https://useast.ensembl.org/info/docs/tools/vep/vep_formats.html). ClinVar significance is just an annotation about the significance of any given variant: https://www.ncbi.nlm.nih.gov/clinvar/docs/clinsig/#clinsig_scv. 

VEP can assign many different consequences to the variants based on the alternative alleles. The concept of "worst" comes into play when you have a lot of different annotations for a given variant. In that case, you often want to check what's the "worst" case scenario for that variant. 

So what this sentence is saying is: for any given variant, look through all the annotations and see what the worst ClinVar significance is. If its "only" benign, then we drop the variant.

**Review by dmiller15 — CHANGES_REQUESTED** [review]

File: docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+| No PoN | TOSCA (PASS+GNOMAD_HARD+DP30)|  37140 |  32411 |  72772 |  0.41 | 0.53 | 0.34 |
+| No PoN | Blacklist TOSCA (PASS+GNOMAD_HARD+DP30)| 36996 |  27029 |  72916 |  0.43 | 0.58 | 0.34 |
+| Broad PoN | TOSCA (PASS+GNOMAD_HARD+DP30)|  37129 |  25406 |  72783 |  0.43 | 0.59 | 0.34 |
+| Broad PoN | Blacklist TOSCA (PASS+GNOMAD_HARD+DP30)| 37004 |  21426 |  72908 |  0.44 | 0.63 | 0.34 |
+| GDC PoN | TOSCA (PASS+GNOMAD_HARD+DP30)|  36995 |  15999 |  72917 |  0.45 | 0.7 | 0.34 |
+| GDC PoN | Blacklist TOSCA (PASS+GNOMAD_HARD+DP30)| 36837 |  13898 |  73075 |  0.46 | 0.73 | 0.34 |
+
+## WXS Filtering
+WGS filtering is generalizable and safe to use for WXS, but WXS just needs a little more. After examining a well-characterized matched tumor-normal WXS set from the OpenPedCan project, we learned the following:
+ - Mutect2 results may have 0 allele depths for some calls. This is explained [here](https://gatk.broadinstitute.org/hc/en-us/articles/360035532252-Allele-Depth-AD-is-lower-than-expected) and has spurred a common sense addition of `AD > 0`
```
[the remark this answers is not in the document]
Comment (migbro) [reply docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md:108]: Ah ok, I can try and move that up///

**Review by dmiller15 — APPROVED** [review]

**Pull request merged**